scrape.py

```diff
@@ -77,6 +134,15 @@ def getStates(item):
 # digestLink is the link to an article
 
 
+def check_states(text):
+    retArr = []
+    for s in stateNames:
+        if s in ' '.join(text):
```

Comment:
if we provide a string in `text` and use `' '.join(text)`, it will add spaces between every character. I don't think we want that.

to check the presence of a state name, we should do something like:

```python
 for s in stateNames:
        if s in text.lower():
            retArr.append(s)
```